### ISSUE TITLE
Option to change the floating button function

### ISSUE THREAD

**Issue by MoneyAllDay** [LiveFastEatTrashRaccoon/RaccoonForLemmy#126]
Option to change the floating button function, for example change it to the "hide read", so we can press it once, instead of having to hold it to press the other functions we want.

I checked all the settings but couldn't find it.

Little note, the mark posts as read on scroll, should mark them as read only after they surpass the top of the screen and not as soon as they appear at the bottom. Maybe I should've opened another suggestion but I didn't wanna spam it 😐.

**Comment by AkesiSeli** [comment]
Yes, they are two separate points but an issue is enough I'll link multiple PR to it just in case!

1. Floating action button (FAB): that is a really cool idea and if I remember well some Lemmy apps have such an option (e.g. Thunder). Unfortunately Raccoon to the current day does not but it would be nice to add it. The issue is that multiple screens have multiple actions linked to the FAB so it is not a "once and for all" choice. I should add multiple items in the settings screens to configure the "preferred FAB action" for home, post detail, community detail, user detail, and so on. A little cumbersome for the user to configure but it saves a lot of clicks once you get it to work.

2. Mark post as read: this is an interesting point and nobody ever complained about it but what you say is true: as soon as they enter the viewport they are marked as read. Probably I should at least wait for them to be completely visible before triggering the action.

**Comment by kolorafa** [comment]
I would prefer to change the + sign to go back to posts, or totallly disable as it colide with the bottom buttons. 

![Image](https://github.com/user-attachments/assets/24e62569-75ed-4726-806b-6833d57678fc)
